codegen/src/next/mod.rs

```diff
@@ -95,11 +92,17 @@ impl Endpoint {
         };
 
         // We'll use the paren span for errors later
-        let paren = params.paren_token;
+        let endpoint_delim_span = &endpoint
+            .meta
+            .require_list()
+            .expect("Params Delimiter")
```

Comment:
I think this expect is okay since the params parsing above will error if the endpoint [meta](https://docs.rs/syn/latest/syn/enum.Meta.html) is not a `MetaList`, but perhaps we should rephrase it?
```suggestion
            .expect("endpoint meta should be a list")
```